.gitignore

```diff
@@ -1,4 +1,5 @@
 node_modules
 dist
 yarn-error.log
-coverage
\ No newline at end of file
+coverage
+.env
```

Comment:
Did you intend to commit this? Can this be removed as it is not relevant to this PR?

Reply:
Yes it's intentional. Would come in handy when we setup `auto`